docs/reference/modules/asset.md

```diff
@@ -77,6 +78,105 @@ modules/@apostrophecms/asset/index.js
 
 To split files within a single `ui/src` folder into multiple bundles, assign each file separately with a property:value pair for each file.
 
+### `devicePreviewMode`
+The `devicePreviewMode` is disabled by default and when activated adds icons to the admin-bar for each of the breakpoints specified in the `screens` object. Clicking on these icons will cause the markup to display as a CSS container of the specified dimensions. Any styling assets handled by the apostrophe build process will be applied as container queries in this display. This means that CSS breakpoint styling added through `<style>` blocks in the template will not change in response to the altered display size. There are also some standard CSS breakpoint declarations that cannot be directly converted into container queries, particularly those targeting the viewport height or those relying on global viewport conditions. You will receive console notifications when declarations that can't be converted are encountered if the `debug` property is set to `true`.
```

Comment:
* Enabled by default
* We need to explain first that we are transpiling to container queries only for preview purposes and we do not alter them outside of preview

Reply:
Changed.